### ISSUE TITLE
The populated path is incorrect

### ISSUE THREAD

**Issue by tetap** [orhanbalci/rough-rs#11]
This is the original: 
![image](https://user-images.githubusercontent.com/57606461/216863048-fc536c98-e05b-4c4f-94c5-fa4721da03c7.png)
My code:
```rust
let svg_path = "M140 20C73 20 20 74 20 140c0 135 136 170 228 303 88-132 229-173 229-303 0-66-54-120-120-120-48 0-90 28-109 69-19-41-60-69-108-69z";
    let options = roughr::core::OptionsBuilder::default()
        .hachure_angle(90.0)
        .hachure_gap(10.0)
        .roughness(0.0)
        .simplification(20.0)
        .fill(roughr::Srgb::new(1.0, 0.0, 0.0).into_format())
        .build()
        .unwrap();
    let generator = rough_piet::KurboGenerator::new(options);
    let path = generator.path::<f32>(svg_path.to_string());
    let sets = path.sets;
    for set in sets.into_iter() {
        std::fs::write(
            format!("test/image/scanline{:?}.txt", set.op_set_type),
            set.ops.to_svg(),
        )
        .unwrap();
    }
```

And this is what you get ↓
FillSketch:
![image](https://user-images.githubusercontent.com/57606461/216863327-ae644df1-d227-4d74-8241-b9938cdfce22.png)

**Comment by tetap** [comment]
![image](https://user-images.githubusercontent.com/57606461/216863507-170b21e2-a75f-4238-ba02-6f45b82376e8.png)

**Comment by tetap** [comment]
![image](https://user-images.githubusercontent.com/57606461/216864401-66aabcae-71f4-47a6-831d-bfff2c72020c.png)

This is the test result of my test case in js

**Comment by orhanbalci** [comment]
Can you please add your js test case code.

**Comment by tetap** [comment]
> roughness

<html>

<head>
  <title>RoughJS sample</title>
  <script src="https://unpkg.com/roughjs@latest/bundled/rough.js"></script>
</head>

<body>
  <h2>RoughJS Path Example (Canvas)</h2>
  <canvas id="canvas" width="800" height="600"></canvas>

  <script>
    const rc = rough.canvas(document.getElementById('canvas'));
    rc.path('M140 20C73 20 20 74 20 140c0 135 136 170 228 303 88-132 229-173 229-303 0-66-54-120-120-120-48 0-90 28-109 69-19-41-60-69-108-69z', { fill: 'blue', hachureAngle: 90, hachureGap: 1,roughness:0 });
  </script>
</body>

</html>

**Comment by tetap** [comment]
> 你能添加你的js测试用例代码吗？

````
<html>

<head>
  <title>RoughJS sample</title>
  <script src="https://unpkg.com/roughjs@latest/bundled/rough.js"></script>
</head>

<body>
  <h2>RoughJS Path Example (Canvas)</h2>
  <canvas id="canvas" width="800" height="600"></canvas>

  <script>
    const rc = rough.canvas(document.getElementById('canvas'));
    rc.path('M140 20C73 20 20 74 20 140c0 135 136 170 228 303 88-132 229-173 229-303 0-66-54-120-120-120-48 0-90 28-109 69-19-41-60-69-108-69z', { fill: 'blue', hachureAngle: 90, hachureGap: 1,roughness:0 });
  </script>
</body>

</html>

````

**Comment by tetap** [comment]
> 

https://heavy-swamp-kilometer.glitch.me/

**Comment by tetap** [comment]
> Can you please add your js test case code.

HI.I wonder if you have time to read it.Unfortunately, I can't solve this problem.

**Comment by orhanbalci** [comment]
This commit fixes problem https://github.com/orhanbalci/rough-rs/commit/46e5c2e47fdc9e56397ec1c51f1fd7cb94e77b1d

**Comment by tetap** [comment]
> 此提交修复了问题[46e5c2e](https://github.com/orhanbalci/rough-rs/commit/46e5c2e47fdc9e56397ec1c51f1fd7cb94e77b1d)

Seems like the version hasn't been released yet?

**Comment by orhanbalci** [comment]
released 0.5.0 on creates.io
